behat.yml

```diff
@@ -0,0 +1,18 @@
+default:
+  suites:
+    standard:
+      paths: ['Features']
+      contexts: [MainContext]
+
+  extensions:
+    Behat\MinkExtension:
+      default_session: selenium2
+      javascript_session: selenium2
+      base_url:  'http://behat-example.qa-tools.io'
```

Comment:
Maybe we should use `http://localhost` as url here and in other setting. It's most unlikely that people will have that domain.

Also in README.md I recommend mentioning steps required to make example operational, like go and replace `base_url` in 2 places with your actual url.